src/main.rs

```diff
@@ -114,8 +107,8 @@ fn check_nixpkgs(nixpkgs_path: &Path) -> validation::Result<ratchet::Nixpkgs> {
         return Ok(Success(ratchet::Nixpkgs::default()));
     }
 
-    let mut nix_file_store = NixFileStore::default();
-    let structure = check_structure(&nixpkgs_path, &mut nix_file_store)?;
+    let mut nix_file_store = nix_file::Store::default();
+    let structure = structure::check(&nixpkgs_path, &mut nix_file_store)?;
```

Comment:
I don't like this change. the user could end up with `Store` instead of `nix_file::Store`, and that makes for unreadable code.

If we want to use module name repetition lint, I suggest setting up pub/private namespaces, and re-exports, such that the end user must use `NixFileStore` and `structure_check`, at which point, it doesn't really make sense.